flickit-assessment-data/src/main/resources/liquibase/changelog-0.36.xml

```diff
@@ -65,4 +65,21 @@
                              constraintName="uq_fac_attribute_value_assessmentresultid_attributeid"
                              deferrable="true" initiallyDeferred="true"/>
     </changeSet>
+
+    <changeSet id="0.36-07" author="Maziyar Gerami">
+        <addColumn tableName="fak_kit_version">
+            <column name="status_version" type="bigint" valueComputed="
+            CASE
+                WHEN status = 0 THEN 0
+                WHEN status = 1 THEN 1
+                WHEN status = 2 THEN -id
+            END"/>
```

Comment:
**Suggestion:** Verify the logic in the new changeSet for adding the status_version column to ensure it handles all potential status values correctly and does not produce unexpected results for statuses other than 0, 1, or 2. [general, importance: 8]
```suggestion
            <column name="status_version" type="bigint" valueComputed="
                CASE
                    WHEN status = 0 THEN 0
                    WHEN status = 1 THEN 1
                    WHEN status = 2 THEN -id
                    ELSE NULL
                END"/>
```